[2022-01-29 12:27:20,659] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-01-17T00:00:00+00:00 [queued]>
[2022-01-29 12:27:20,821] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-01-17T00:00:00+00:00 [queued]>
[2022-01-29 12:27:20,821] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 12:27:20,821] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-01-29 12:27:20,821] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 12:27:20,940] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-01-17 00:00:00+00:00
[2022-01-29 12:27:20,988] {standard_task_runner.py:52} INFO - Started process 3388 to run task
[2022-01-29 12:27:21,050] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2019-01-17T00:00:00+00:00', '--job-id', '252', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpp7k56hd4', '--error-file', '/tmp/tmp7hmumvu8']
[2022-01-29 12:27:21,061] {standard_task_runner.py:77} INFO - Job 252: Subtask local_to_gcs_task
[2022-01-29 12:27:21,576] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-01-17T00:00:00+00:00 [running]> on host 85e1117cfb12
[2022-01-29 12:27:22,014] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-17T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-17T00:00:00+00:00
[2022-01-29 12:27:23,722] {python.py:175} INFO - Done. Returned value was: None
[2022-01-29 12:27:23,746] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20190117T000000, start_date=20220129T122720, end_date=20220129T122723
[2022-01-29 12:27:23,866] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-29 12:27:23,987] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
